hack/update-bazel.sh

```diff
@@ -70,7 +70,9 @@ gazelle fix \
     -mode=fix \
     -repo_root "${KUBE_ROOT}" \
     "${KUBE_ROOT}" \
-    2>&1 | grep -Ev "vendor/github.com/seccomp/libseccomp-golang/seccomp(_internal)?.go: pkg-config not supported: #cgo pkg-config: libseccomp" || (exit "${PIPESTATUS[0]}")
+    2>&1 | grep -Ev "vendor/github.com/seccomp/libseccomp-golang/seccomp(_internal)?.go: pkg-config not supported: #cgo pkg-config: libseccomp" \
```

Comment:
see https://github.com/kubernetes/kubernetes/pull/79502/files#diff-fa58360a7125fddab9491c80bb3f1a87R72

Reply:
@aojea  Thank  you. This output is a bit annoying.

https://github.com/kubernetes/kubernetes/issues/79499#issuecomment-614459419